docs/Node-for-Formula-Authors.md

```diff
@@ -91,10 +91,10 @@ Installing a standard Node module based formula would look like this:
 require "language/node"
 
 class Foo < Formula
-  desc "..."
-  homepage "..."
+  desc "Description"
```

Comment:
Same here

Reply:
Same as? Which previous comment does this refer to?